### DISCUSSION TITLE
Labeling Question

### DISCUSSION THREAD

**Discussion by blurredrain** [ultralytics/yolov5#8946] [Q&A] [answered]
Hi,
Sorry for this probably annoying question but I found only a few answers which differ from each other.

For YOLOv5, is it better to label an object like it would be fully visible, or only to label what you can see.

In my dataset are only persons, but 10% of them are behind a window or fence so nearly exact 50% of them (legs, feet) are occluded.

![Screenshot_20220813-022934_Chrome](https://user-images.githubusercontent.com/111154395/184460699-0d366c9d-6238-4307-8b63-6285ca65c61f.jpg)
How would you label them?

**Comment by glenn-jocher** [comment, marked as answer]
@blurredrain it's up to you. You should label your data the way you want your model to predict during deployment. If you want it to predict people behind occlusions then you should label that way, or if you want it to only box visible people then you should go with the left example.

Between the two you'll probably have a simpler time annotating the left way.

**Reply by blurredrain** [reply]
Thank you :)
